test/test_notebook.py

```diff
@@ -92,6 +92,14 @@ def test_hierarchy_flow_using_select_speaker(save=False):
     run_notebook("agentchat_hierarchy_flow_using_select_speaker.ipynb", save=save)
 
 
+@pytest.mark.skipif(
+    skip or not sys.version.startswith("3.10"),
+    reason="do not run if openai is not installed or py!=3.10",
+)

```

Comment:
Shall we balance the load by running in 3.11 for this test?

Reply:
Yes